patches/minecraft/net/minecraft/entity/EntityLivingBase.java.patch

```diff
@@ -145,7 +153,15 @@
          super.func_70071_h_();
  
          if (!this.field_70170_p.field_72995_K)
-@@ -2000,4 +2029,39 @@
+@@ -1828,6 +1858,7 @@
+ 
+     public void func_70078_a(Entity p_70078_1_)
+     {
++        if(!net.minecraftforge.event.ForgeEventFactory.canMountEntity(this, p_70078_1_, true)){ this.func_70101_b(field_70126_B, field_70127_C); return; }
```

Comment:
Why is retation needed? Also could make it simpler by moving that to the event factory to.